packages/lib/models/Note.js

```diff
@@ -272,8 +272,13 @@ class Note extends BaseItem {
 				let bProp = b[order.by];
 				if (typeof aProp === 'string') aProp = aProp.toLowerCase();
 				if (typeof bProp === 'string') bProp = bProp.toLowerCase();
-				if (aProp < bProp) r = +1;
-				if (aProp > bProp) r = -1;
+
+				if (order.by == 'title' && Setting.value('titleNaturalSort')) {
```

Comment:
New code should use strict equality `===`

Reply:
Thanks, I've changed all `==` to `===`